dot/network/host_test.go

```diff
@@ -548,14 +554,20 @@ func TestStreamCloseEOF(t *testing.T) {
 
 	stream, err := nodeA.host.send(addrInfoB.ID, nodeB.host.protocolID, testBlockReqMessage)
 	require.NoError(t, err)
-	require.False(t, handler.exit)
 
 	err = stream.Close()
 	require.NoError(t, err)
 
-	time.Sleep(TestBackoffTimeout)
+	timeout := time.NewTimer(TestBackoffTimeout)
```

Comment:
nit rename to `timer`

Reply:
I don't mind this naming.

Reply:
I guess it could be a *timeout timer* as well 😄